### PR TITLE
Improve "rasied file descriptor limit..." log

### PR DESCRIPTION
**Pull request by terencechain**
Minor improvement to the log

`[2022-07-01 08:32:50]  INFO Raised fd limit to 10240 from 61440`
👇🏼 
`[2022-07-01 08:57:42]  INFO main: Raised file descriptor limit to 10240 from 61440`

### CODE REVIEW

**Comment by rkapka** [comment]
`Raised file descriptor limit to 10240 from 61440` - in this example we are not raising. Maybe change `Raised` to `Changed`?

File: runtime/fdlimits/fdlimits.go [hunk 1]
```diff
@@ -2,9 +2,11 @@ package fdlimits
 
 import (
 	"github.com/ethereum/go-ethereum/common/fdlimit"
-	log "github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus"
 )
 
+var log = logrus.WithField("prefix", "main")
```
[the remark this answers is not in the document]
Comment (rkapka) [reply runtime/fdlimits/fdlimits.go:8]: `fdlimits`?

**Comment by terencechain** [comment]
> `Raised file descriptor limit to 10240 from 61440` - in this example we are not raising. Maybe change `Raised` to `Changed`?

Thanks! Ready again

File: runtime/fdlimits/fdlimits.go [hunk 2]
```diff
@@ -20,6 +22,6 @@ func SetMaxFdLimits() error {
 	if err != nil {
 		return err
 	}
-	log.Infof("Raised fd limit to %d from %d", raisedVal, curr)
+	log.Infof("Updated file descriptor limit to %d from %d", raisedVal, curr)
```
Comment (prestonvanloon) [inline runtime/fdlimits/fdlimits.go:25]: Could we push this to debug? I notice it on many commands and it doesn't seem like meaningful information.

Comment (terencechain) [reply runtime/fdlimits/fdlimits.go]: Good point!

**Review by prestonvanloon — APPROVED** [review]

**Pull request merged**